src/schema/bioscales.yaml

```diff
@@ -28,8 +31,10 @@ default_curi_maps:
 
 
 imports:
-  - linkml:types
+  - basic_slots
   - core
+  - linkml:types
```

Comment:
Is an explicit `linkml:types` import necessary? I ask because `basic_slots` imports `linkml:types` already

Reply:
good point